LDMP/gui/WidgetMain.ui

```diff
@@ -66,6 +72,9 @@
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_sort">
+            <property name="toolTip">
+             <string>Sort the datasets using the selected property.</string>
```

Comment:
This does not seem to work. I don't see this tooltip on my QGIS after having pulled in the latest changes